### PR TITLE
Bump CL to v1

### PR DESCRIPTION
**Pull request by jcantrill**
This PR bumps CL to v1

https://jira.coreos.com/browse/LOG-370

### CODE REVIEW

File: hack/cr-aws.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-apiVersion: "logging.openshift.io/v1alpha1"
-kind: "ClusterLogging"
-metadata:
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"metadata:
```
Comment (richm) [inline hack/cr-aws.yaml]: this doesn't look right

File: pkg/apis/logging/v1/types.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package v1alpha1
+package v1
 
 import (
 	"github.com/openshift/elasticsearch-operator/pkg/apis/elasticsearch/v1alpha1"
```
Comment (richm) [inline pkg/apis/logging/v1/types.go]:
```
	v1 "k8s.io/api/core/v1"
```
do you need to change this import to `corev1` as in `zz_generated.deepcopy.go`?

Comment (jcantrill) [reply pkg/apis/logging/v1/types.go]: only if there is a conflict and compile error.  That file is generated so they have taken the liberty of using a naming convention

File: hack/cr-aws.yaml (hunk 1 above)
Comment (jcantrill) [reply hack/cr-aws.yaml]: fixed

**Comment by richm** [comment]
can this be merged independent of https://github.com/openshift/elasticsearch-operator/pull/106?

**Pull request merged**